Rakefile

```diff
@@ -1,8 +1,10 @@
 require 'bundler'
 Bundler::GemHelper.install_tasks
 
-require 'rspec/core/rake_task'
-RSpec::Core::RakeTask.new(:spec)
+unless ENV['RACK_ENV'] == 'production'
+  require 'rspec/core/rake_task'
+  RSpec::Core::RakeTask.new(:spec)
 
-task :test => :spec
-task :default => :spec
\ No newline at end of file
+  task :test => :spec
```

Comment:
Yeah, I'm planning on doing a separate PR that fixes Rubocop issues. This is forked code that was in bad shape and I'm slowly but surely getting it in good shape.